decidim-core/config/locales/en.yml

```diff
@@ -6,7 +6,6 @@ en:
         delete_reason: Reason to delete your account
       group:
         about: About
-        avatar: Avatar
```

Comment:
Are you sure we want to delete this? Isn't this used somewhere?

Reply:
Could you please re-add it? The master branch has it and there's no error at the test suite. I'll keep an eye if there's an error here.